site/content/en/_index.html

```diff
@@ -0,0 +1,22 @@
++++
+title = "CVAT"
+linkTitle = "CVAT"
++++
+
+{{< blocks/section height="full" color="docs" >}}
+
+<section id="temporary-page" class="mx-auto text-center py-5">
+  <div class="py-4">
+	<i class="fas fa-tools fa-7x"></i>
+  </div>
+  <div class="py-4">
+	<h1 class="text-center">This page is in development.</h1>
+  </div>
+  <div class="py-4">
+    <h3 class="text-center">
+	    Visit our <a href="https://github.com/openvinotoolkit/cvat">GitHub</a> repository.
+	</h3>
+  </div>
+</section>
+
+{{< /blocks/section >}}
```

Comment:
I would suggest redirecting from this page to About page.  I think it is a good candidate to be default page.

Reply:
And add link to GitHub repository from the header